src/components/AppNavigationItem/AppNavigationItem.vue

```diff
@@ -156,7 +156,9 @@ Just set the `pinned` prop.
 
 		<!-- Counter and Actions -->
 		<div v-if="hasUtils" class="app-navigation-entry__utils">
-			<slot name="counter" />
+			<div class="app-navigation-entry__counter">
```

Comment:
v-if counter?

Reply:
Having this container will lead to two element having the `app-navigation-entry__counter` class:
```
<div class="app-navigation-entry__counter">
   <div data-v-794921ed="" data-v-9941="" class="app-navigation-entry__counter">
		99+
	</div>
</div>
```
I think this doesn't make sense. And it isn't necessary anyway. See https://github.com/nextcloud/nextcloud-vue/pull/1897/files#r623671032